<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabJobType">
      <attribute name="title">
       <string>Тип работы</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="13" column="0">
        <widget class="QCheckBox" name="chkShowOnlyPayed">
         <property name="text">
          <string>Отображать только оплаченные</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblListContext">
         <property name="text">
          <string>Контекст печати списка</string>
         </property>
         <property name="buddy">
          <cstring>edtListContext</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblActionDateChanger">
         <property name="text">
          <string>Модификатор даты действия</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lblDurationVisibility_2">
         <property name="text">
          <string>Способ выбора номерка</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSpinBox" name="edtDurationVisibility"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblRegionalCode">
         <property name="text">
          <string>&amp;Региональный код</string>
         </property>
         <property name="buddy">
          <cstring>edtRegionalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="cmbTicketAssignWay">
         <item>
          <property name="text">
           <string>По времени</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>По УЕТ</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="edtRegionalCode"/>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="edtTicketDuration"/>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="cmbActionDateChanger">
         <item>
          <property name="text">
           <string>Не меняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Дата окончания</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="cmbActionPersonChanger">
         <item>
          <property name="text">
           <string>Не меняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Пользователь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Назначивший действие</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ответственный за событие</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="cmbActionStatusChanger">
         <item>
          <property name="text">
           <string>Не меняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Начато</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ожидание</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Закончено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Отменено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Без результата</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Назначено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Не предусмотрено</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblDurationVisibility">
         <property name="text">
          <string>Продолжительность видимости расписания</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="edtListContext"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblActionStatusChanger">
         <property name="text">
          <string>Модификатор статуса действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionStatusChanger</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblTicketDuartion">
         <property name="text">
          <string>Длительность номерка</string>
         </property>
         <property name="buddy">
          <cstring>edtTicketDuration</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblActionPersonChanger">
         <property name="text">
          <string>Модификатор исполнителя действия</string>
         </property>
         <property name="buddy">
          <cstring>cmbActionPersonChanger</cstring>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lblJobStatusModifier">
         <property name="text">
          <string>Модификатор состояния работы</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QComboBox" name="cmbJobStatusModifier">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выбранное значение будет проставляться автоматически в поле Состояние при открытии работы в журнале Выполнение работ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Не задано</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Выполняется</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Закончено</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabActionTypes">
      <attribute name="title">
       <string>Действия</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblJobTypeActions"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabQuota">
      <attribute name="title">
       <string>Квотирование</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblQuota"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtListContext</tabstop>
  <tabstop>cmbActionStatusChanger</tabstop>
  <tabstop>cmbActionPersonChanger</tabstop>
  <tabstop>cmbActionDateChanger</tabstop>
  <tabstop>edtTicketDuration</tabstop>
  <tabstop>tblJobTypeActions</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
